docs/Developer/ClassDiagrams/packages.puml

```diff
@@ -0,0 +1,100 @@
+@startuml package UML
+
+/' set up line '/
+skinparam linetype ortho
+/' hide the class members '/
+hide members
+
+/' packages '/
+package Core{
+	/' class '/
+   class Simulator
+   class CPUSpikingModel implements Model
+	class GPUSpilingModel implements Model
+	class Operations
+	class OperationManager
+	class ParameterManager
+
+	/' class relationships '/
+	Model -left-o Simulator 
+	Operations o-right- OperationManager   
```

Comment:
`OperationsManager` contains `Operations`, right?

Reply:
From the hand_draw diagram, it looks like the Operations contains OperationsManager? The diamond of UML arrow in this diagram is next to Operations.

Reply:
You found an error in the hand drawn diagram that nobody has noticed! That's great; and we can fix it here.